homeassistant/components/ambient_network/sensor.py

```diff
@@ -0,0 +1,310 @@
+        suggested_display_precision=0,
+    ),
+    SensorEntityDescription(
+        key=TYPE_BAROMABSIN,
+        translation_key="absolute_pressure",
+        native_unit_of_measurement=UnitOfPressure.INHG,
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_BAROMRELIN,
+        translation_key="relative_pressure",
+        native_unit_of_measurement=UnitOfPressure.INHG,
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_CO2,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        device_class=SensorDeviceClass.CO2,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_DAILYRAININ,
+        translation_key="daily_rain",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        device_class=SensorDeviceClass.PRECIPITATION,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_DEWPOINT,
+        translation_key="dew_point",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_FEELSLIKE,
+        translation_key="feels_like",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_HOURLYRAININ,
+        translation_key="hourly_rain",
+        native_unit_of_measurement=UnitOfVolumetricFlux.INCHES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PRECIPITATION_INTENSITY,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_HUMIDITY,
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LASTRAIN,
+        translation_key="last_rain",
+        icon="mdi:water",
+        device_class=SensorDeviceClass.TIMESTAMP,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LIGHTNING_PER_DAY,
+        translation_key="lightning_strikes_per_day",
+        icon="mdi:lightning-bolt",
+        native_unit_of_measurement="strikes",
+        state_class=SensorStateClass.TOTAL,
+        suggested_display_precision=0,
```

Comment:
I don't think this is needed, I've never seen half a strike

Reply:
Removed.